docs/docs/noir/concepts/traits.md

```diff
@@ -145,9 +145,15 @@ fn main() {
 }
 ```
 
+### Generic Implementations With Where Clauses
+
+Where clauses can be placed on struct implementations themselves to restrict generics.
+
+For example, while `impl<T> Foo` 
```

Comment:
Unfinished documentation - looks like the block below was meant to be added here?

Reply:
I had actually just decided to move the documentation below, but forgot to remove these old changes.